src/Hunk/UnifiedHunk/__test__/UnifiedChange.test.js

```diff
@@ -0,0 +1,42 @@
+import renderer from 'react-test-renderer';
+import {shallow} from 'enzyme';
+import {partialRight, get} from 'lodash';
+import UnifiedChange from '../UnifiedChange';
+
+const defaultChange = {
+    type: 'edit',
+    content: 'text',
+};
+
+const pickId = partialRight(get, 'content');
+
+describe('UnifiedChange', () => {
+    test('renders correctly', () => {
+        expect(renderer.create(
+            <UnifiedChange
+                selected={false}
+                change={defaultChange}
+                generateAnchorID={pickId}
+            />
+        ).toJSON()).toMatchSnapshot();
+    });
+
+    test('gutterEvents should be called with side', () => {
+        const onClick = jest.fn();
+        const gutterEvents = {onClick};
+        const wrapper = shallow(
+            <UnifiedChange
+                selected={false}
+                change={defaultChange}
+                generateAnchorID={pickId}
+                gutterEvents={gutterEvents}
+            />
+        );
+        // TODO diff gutter should be called with {change: defaultChange, side: 'old' & 'new'}
+        wrapper.find('.diff-gutter').at(0).simulate('click');
+        expect(onClick).toBeCalledWith({change: defaultChange, side: undefined});
```

Comment:
IMO we should test `side: 'old'` here so test case fails and we write code to fix it later

Reply:
It is complicated to decide what is right at this moment since `UnifiedChange` currently does not have the concept of `side`.
Maybe we will change the Component itself a lot.